README.md

```diff
@@ -8,55 +10,44 @@ This buildpack will participate if all the following conditions are met
 
-```shell
-docker run --rm -p 8080:8080 my-app
-```
-
-Check the logs and verify the OpenTelemetry Agent was included.
+## Usage
 
-```log
-Picked up JAVA_TOOL_OPTIONS: -Djava.security.properties=/layers/paketo-buildpacks_bellsoft-liberica/java-security-properties/java-security.properties -XX:+ExitOnOutOfMemoryError -javaagent:/layers/paketo-buildpacks_opentelemetry/opentelemetry-java/opentelemetry-javaagent.jar -XX:ActiveProcessorCount=3 -XX:MaxDirectMemorySize=10M -Xmx3537385K -XX:MaxMetaspaceSize=98270K -XX:ReservedCodeCacheSize=240M -Xss1M -XX:+UnlockDiagnosticVMOptions -XX:NativeMemoryTracking=summary -XX:+PrintNMTStatistics
+Once you enable the OpenTelemetry buildpack at build-time, you can configure it at run-time via environment variables, as described in the [project documentation](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/) or by passing configuration properties to the container via a binding.
 
-[otel.javaagent 2022-06-28 21:16:29:365 +0000] [main] INFO io.opentelemetry.javaagent.tooling.VersionLogger - opentelemetry-javaagent - version: 1.15.0
-```
+By default, the following configuration is applied to the OpenTelemetry Java Agent at run time.
 
-## Usage
+* `OTEL_JAVAAGENT_ENABLED=false`
+* `OTEL_METRICS_EXPORTER=none`
 
-Once you enable the OpenTelemetry buildpack, you can configure it at run-time via environment variables, as described in the [project documentation](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/).
+When using a binding to configure the agent, properties are expected to be passed via a config tree format,
+where each key is the name of a file and the value its content. Keys can follow the environment variable format or the
+system property format, as described in the [project documentation](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/).
 
 ## Bindings
 
 The buildpack optionally accepts the following bindings:
 
-### Type: `dependency-mapping`
+### Type: `opentelemetry`
 
 | Key                   | Value   | Description                                                                                       |
 | --------------------- | ------- | ------------------------------------------------------------------------------------------------- |
-| `<dependency-digest>` | `<uri>` | If needed, the buildpack will fetch the dependency with digest `<dependency-digest>` from `<uri>` |
+| `<otel-property-key>` | `<otel-property-value>` | Properties are expected to be passed via a config tree format, where each key is the name of a file and the value its content. That's the default behavior when passing key/value pairs via a Kubernetes Secret. Keys can follow the environment variable format or the system property format, as described in the [project documentation](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/).  |
```

Comment:
Same note here. This might be too much info and distracting from the purpose. My suggestion:

```
Binding key/values map directly to Java Open Telemetry agent configuration properties. Keys can follow the environment variable format or the system property format, as described in the [project documentation](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/).

Reply:
Good point, I replaced it with your suggestion, thanks.